oracle/oracles/remote.go

```diff
@@ -24,6 +26,10 @@ func NewRemoteOracle(addr string) oracle.Oracle {
 	}
 }
 
+func (t *remoteOracle) IsExpired(beginMs uint64, TTL uint64) bool {
+	return uint64(time.Now().UnixNano()/int64(time.Millisecond)) >= (beginMs + TTL)
```

Comment:
Is it correct to use local timestamp here? I noticed that beginMs was came from remote TSO.